R/helpers_jags.R

```diff
@@ -68,6 +68,11 @@ h_jags_join_models <- function(model1, model2) {
   assert_class(body(model1), "{")
   assert_class(body(model2), "{")
 
+  # workaround needed to avoid bugs related to covr-injected code
+  if (h_covr_active()) {
+    body(model2) <- h_covr_detrace(body(model2))
+  }
```

Comment:
This is what it looks like when used within `h_jags_join_models` to avoid injection of covr traces while building our model code.